src/main/java/com/iota/iri/network/TransactionRequester.java

```diff
@@ -53,8 +52,8 @@ public boolean clearTransactionRequest(Hash hash) {
         }
     }
 
-    public void requestTransaction(Hash hash) throws Exception {
-        if (!snapshotProvider.getInitialSnapshot().hasSolidEntryPoint(hash) && !TransactionViewModel.exists(tangle, hash)) {
+    public void requestTransaction(Hash hash){
```

Comment:
Will be done in a later PR.

Reply:
Guess it is according to the rules of what is allowed since it was an existing method.
But you can add it still just to make the code a little better.

Anyhows, you inspired me to add the `"WhitespaceAround"` rule to checkstyle now. So mind adding that extra space before the `{`? 
It will have to happen once the rule is added anyhows :-)